doc/source/examples.rst

```diff
@@ -103,3 +103,77 @@ By default, ``frame_to_hyper`` and ``frames_to_hyper`` will fully drop and reloa
    pantab.frame_to_hyper(df, "example.hyper", table="animals", table_mode="a")
 
 Please note that ``table_mode="a"`` will create the table(s) if they do not already exist.
+
+Inserting New Data into Existing Tables
+---------------------------------------
+
+In addition to append, ``table_mode="i"`` can be used to combine a df with an existing table, only inserting new records. A unique key ``table_key="column"`` must be supplied when using this method.
```

Comment:
I think calling this `insert` is a little vague; actually there is a outstanding PR in pandas to implement this for SQL engines where they use the terminology `upsert_overwrite` and `upsert_ignore`.

https://github.com/pandas-dev/pandas/pull/29636/files

This PR goes just for `ignore` which is OK, but don't want to lose sight of an `overwrite` feature so maybe want to word this differently